randovania/games/am2r/json_data/Main Caves.txt

```diff
@@ -375,8 +377,8 @@ Extra - minimap_data: [{'x': 47, 'y': 16}, {'x': 47, 'y': 17}]
               Infinite Bomb Jumping (Advanced) and Can Use Bombs
               # Intended spider ball way
               Can Use Bombs and Can Use Spider Ball
-              # Shinespark up
-              Speed Booster and Shinesparking Tricks (Intermediate) and Disabled Entrance Rando
+              # Shinespark Up (Stored from previous room)
+              Speed Booster and Shinesparking Tricks (Advanced) and Disabled Entrance Rando
```

Comment:
Hm, not sure if this should be advanced tbh. Since there's a really long place from right before the room where you can charge the shinespark, so it's just
- lay bombs to destroy blocks
- walk
- crouch
- go into tunnel
- position yourself., and spark up.

The hardest part tbh is knowing where to spark, since there's only tile where you will hit it. But still, due to no chaining being required, this is way easier than i.e. the GT spark, or the a3 secret

Reply:
I would say yes to intermediate IF the bomb blocks didn't respawn so quickly. 
If the `A5 space jump trial access` shinespark is intermediate I have to push that this is definitely more technical.